lldb/test/Shell/ObjectFile/XCOFF/basic-info.yaml

```diff
@@ -13,7 +13,7 @@ FileHeader:
   MagicNumber:     0x1F7
   NumberOfSections: 1 
   CreationTime:    000000000
-  Flags:           0x0000
+  Flags:           0x0002
```

Comment:
Not sure if we're allowed comments in this YAML. But could you at least confirm to me here (in reply to this comment I mean), what 2 means? Is this the EXEC flag?

I guess that this test started failing because you actually made the parsing better. Is that the case?

Reply:
Yes, that is the exact reason for this change. A 2 here implies that it is an exec.